libs/api/src/main/java/de/cyface/api/ClassifiedSegmentRetriever.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2021 Cyface GmbH
+ *
+ * This file is part of the Cyface Data Collector.
+ *
+ * The Cyface Data Collector is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Cyface Data Collector is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the Cyface Data Collector. If not, see <http://www.gnu.org/licenses/>.
+ */
+package de.cyface.api;
+
+import java.time.OffsetDateTime;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import de.cyface.api.model.ClassifiedSegment;
+import de.cyface.api.model.Geometry;
+import de.cyface.model.Modality;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.mongo.MongoClient;
+
+/**
+ * Loads classified segments from the database for a vert.x context.
+ *
+ * @author Armin Schnabel
+ */
+@SuppressWarnings("unused") // Part of the API
+public class ClassifiedSegmentRetriever {
```

Comment:
I added this class to be able to load `segments` from the database and provide them to the web-app